.examples/docker-compose/insecure/postgres/fpm/docker-compose.yml

```diff
@@ -27,10 +32,20 @@ services:
     ports:
       - 8080:80
     volumes:
+      # volumes must copy-past volume section from app with readonly 
+      - nextcloud-data:/var/www/html/data:ro
+      - nextcloud-config:/var/www/html/config:ro
```

Comment:
nginx does not need acces to `nextcloud-data` or `nextcloud-config`. `nextcloud-theme` is deprecated.
Could you move this to a section in the README? I think by default only one volume should be used, but you could leaf them commented as an example.